### PR TITLE
docs: godoc for package and example

### PR DESCRIPTION
**Pull request by DenBeke**

### CODE REVIEW

File: example_test.go [hunk 1]
```diff
@@ -0,0 +1,32 @@
+package scalewaysdkgo
+
+import (
+	"fmt"
+
+	"github.com/scaleway/scaleway-sdk-go/api/instance/v1"
+	"github.com/scaleway/scaleway-sdk-go/scw"
+)
+
+func Example_apiClient() {
```
Comment (kindermoumoute) [inline example_test.go]: Let's keep things simple: no error handling, no pretty display.

Also we could add few comment on what does each line.

```Go
func Example_apiClient() {
	// create a Scaleway client
 	client, _ := scw.NewClient(
		scw.WithAuth("ACCESS_KEY", "SECRET_KEY"), // Get your credentials at https://console.scaleway.com/account/credentials
	)

	// get instance SDK
 	instanceApi := instance.NewApi(client)

	// do a request
 	response, _ := instanceApi.ListServers(&instance.ListServersRequest{
		Zone: scw.ZoneFrPar1,
	})

	fmt.Println(response)
 }
```

File: example_test.go [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+	instanceApi := instance.NewApi(client)
+
+	response, err := instanceApi.ListServers(&instance.ListServersRequest{
+		Zone: scw.ZoneFrPar1,
+	})
+	if err != nil {
+		panic(err)
+	}
+
+	for _, server := range response.Servers {
```
Comment (kindermoumoute) [inline example_test.go]: Is it required to have valid Go code in example? ie. can we not use response here?

Comment (DenBeke) [reply example_test.go]: The example should compile.

File: example_test.go (hunk 1 above)
Comment (DenBeke) [reply example_test.go]: I just copied the file from the `README.md` file.

Comment (kindermoumoute) [inline example_test.go]: - create client (simple)
- list servers
- create load balancer

File: example_test.go [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+	// Create SDK objects for specific Scaleway Products
+	instance := instance.NewApi(client)
+	lb := lb.NewApi(client)
+
+	// Start using the SDKs
+	_, _ = instance, lb
+
+}
+
+func Example_getServers() {
```
Comment (kindermoumoute) [inline example_test.go]: list servers*

**Pull request merged**